src/charts/template/Component.js

```diff
@@ -15,6 +15,10 @@ class {{pascalCase componentName}} extends React.Component {
         /**
          * TODO: Add your propTypes here!
          */
+        /**
+         *
+         */
+        : PropTypes.,
```

Comment:
?

Reply:
This is to make it easy to add the proptypes and comments (note this is the template for the charts)